support-frontend/assets/components/subscriptionCheckouts/subscriptionSubmitButtons.jsx

```diff
@@ -29,8 +34,24 @@ type PropTypes = {|
   billingPeriod: BillingPeriod,
   validateForm: Function,
   formIsValid: Function,
+  allErrors: FormError<FormField>[],
 |};
 
+type ErrorSummaryPropTypes = {
+  errors: Array<Object>,
+}
+
+const ErrorSummary = (props: ErrorSummaryPropTypes) => (
+  <div className="component-form-error__border">
+    <p className="component-form-error__heading">There is a problem</p>
+    {props.errors.map(error => (
+      <div className="component-form-error__summary-error">
+        {error.message}
```

Comment:
Should this be a `<p>` or `<li>` rather than a `<div`?

Reply:
Thanks, I have updated this to `<ul><li></li></ul>`